### PR TITLE
[FIX] Shows name under full name section.

### LINKED ISSUE #21129: User info shows username instead of full name
### Description:

The “Full Name” that’s shown when clicking “View Profile” on a user in a channel shows only their *username* instead of their *full name.* The correct full name is shown in the admin panel user info but not on the user front end.

I have tested or verified this in both the desktop app and the web ui.

### Steps to reproduce:

1. Click on a username in a channel
2. Click "See full profile"
3. A panel opens called "User Info".  Under "Full Name" it shows the username instead of the full name.

### Expected behavior:

The "*Full Name*" section should show the user's *full name*.

### Actual behavior:

The "*Full Name*" section wrongly shows the user's *username*.

Screen Shot 2021-03-15 at 4 12 18 PM

And here is a view of the same user from the admin panel which shows the correct *full name*:

Screen Shot 2021-03-15 at 4 14 46 PM

### Server Setup Information:

- Rocketchat Version 3.12.1
- Ubuntu 20.04
- Installed via snaps
- Desktop App v3.1.1 / All Browsers

### Version of Rocket.Chat Server

- Rocketchat Version 3.12.1

### Client Setup Information

- All browsers
- OS X / Windows 10

### PR DESCRIPTION
**Pull request by Darshilp326**
## Proposed changes (including videos or screenshots)

Shows name irrespective of setting 'Show real names' is turned on/off.

## Issue(s)
Closes #21129

### CODE REVIEW

File: client/views/room/contextualBar/UserInfo/index.js [hunk 1]
```diff
@@ -172,7 +172,7 @@ export const UserInfoWithData = React.memo(function UserInfoWithData({ uid, user
 		} = user;
 		return {
 			_id,
-			name: showRealNames ? name : username,
+			name,
```
Comment (ggazzo) [inline client/views/room/contextualBar/UserInfo/index.js:175]: check the warning and the possibility of 'name' being ''

**Review by ggazzo — CHANGES_REQUESTED** [review]

File: client/views/room/contextualBar/UserInfo/index.js (hunk 1 above)
Comment (Darshilp326) [reply client/views/room/contextualBar/UserInfo/index.js:175]: Can you explain it more?

> check the warning and the possibility of 'name' being ''

Comment (ggazzo) [reply client/views/room/contextualBar/UserInfo/index.js:175]: if the name is not provided ... we gonna see something like: 
image

Comment (Darshilp326) [reply client/views/room/contextualBar/UserInfo/index.js:175]: So should we display username when the name is empty?

Comment (Darshilp326) [reply client/views/room/contextualBar/UserInfo/index.js:175]: Hello! Can you please help to solve this issue?

**Comment by dougfabris** [comment]
Hi there, thanks for the contribution! 🚀 💯 

Closing this PR because it's stale now and it was already fixed in another one.

---

**Questions? Help needed? Feature Requests?** 

- Join our [Open Server](https://open.rocket.chat/) in the #support channel and feel free to raise a question
- Join our [Community Forum](https://forums.rocket.chat/) and search/create a post there

**Pull request closed without merging**